INSTALL.md

```diff
@@ -22,6 +22,7 @@ Prerequisites:
 * sqlite3 library >= 3.8.3 or PostgreSQL database
 * pkg-config
 * libical >= 1.0.0
+* gpgme
```

Comment:
Thanks for the patch.

```suggestion
* libgpgme
```

I think this is better, at least to match gvm-libs.

I don't think we need a version number until we add one to the cmake check.

Reply:
Thanks for your suggestion! I have amended the commit.

Reply:
> I have amended the commit.

That seems to have been a bad idea, unfortunately by the time i realised that i had already discarded the only copy of the original commit i had access to from home over the holidays.

Sorry if this causes you extra work.

Reply:
No problem, thanks.